main.go

```diff
@@ -180,6 +180,7 @@ func main() {
 	nsGatherers := make(prometheus.Gatherers, 0)
 
 	flag.IntVar(&opts.ListenPort, "listen-port", 4040, "HTTP port to listen on")
+	flag.StringVar(&opts.Parser, "parser", "text", "NGINX access log format parser. One of: [text, json]")
```

Comment:
The flags need special consideration in the `LoadConfigFromFlags` function, otherwise they will be ignored -- especially since the `parser.New` will only respect the config (which is either loaded from a file, or explicitly constructed from the flags using said function).

https://github.com/martin-helmich/prometheus-nginxlog-exporter/blob/dabff828967754f2fa5e9f1319ce9c95cb5ec66d/config/loader_flags.go#L11-L19

Reply:
Oh, of course, I didn't notice the `LoadConfigFromFlags` function. Thanks a lot.